### ISSUE TITLE
Dead Lock?

### ISSUE THREAD

**Issue by JohnPark234** [MinimallyCorrect/TickThreading#467]
Definitely deadlocked: 
"Entities in Overworld/0 - 7" Id=50 BLOCKED on me.nallar.tickthreading.util.concurrent.NativeMutex@794a2559 owned by "Entities in Overworld/0 - 5" Id=48
    at me.nallar.tickthreading.minecraft.TickManager.lock(TickManager.java:433)
    -  blocked on me.nallar.tickthreading.util.concurrent.NativeMutex@794a2559
    -  locked me.nallar.tickthreading.util.concurrent.NativeMutex@7f4c8759
    at me.nallar.tickthreading.minecraft.tickregion.TileEntityTickRegion.doTick(TileEntityTickRegion.java:59)
    at me.nallar.tickthreading.minecraft.tickregion.TickRegion.run(TickRegion.java:42)
    at me.nallar.tickthreading.minecraft.ThreadManager$2.run(ThreadManager.java:95)
    at me.nallar.tickthreading.minecraft.ThreadManager$1.run(ThreadManager.java:41)
    at me.nallar.tickthreading.minecraft.ThreadManager$ServerWorkThread.run(ThreadManager.java:167)

"Entities in Overworld/0 - 5" Id=48 BLOCKED on me.nallar.tickthreading.util.concurrent.NativeMutex@7f4c8759 owned by "Entities in Overworld/0 - 7" Id=50
    at me.nallar.tickthreading.minecraft.TickManager.lock(TickManager.java:436)
    -  blocked on me.nallar.tickthreading.util.concurrent.NativeMutex@7f4c8759
    -  locked me.nallar.tickthreading.util.concurrent.NativeMutex@794a2559
    -  locked me.nallar.tickthreading.util.concurrent.NativeMutex@539d9744
    at me.nallar.tickthreading.minecraft.tickregion.TileEntityTickRegion.doTick(TileEntityTickRegion.java:59)
    at me.nallar.tickthreading.minecraft.tickregion.TickRegion.run(TickRegion.java:42)
    at me.nallar.tickthreading.minecraft.ThreadManager$2.run(ThreadManager.java:95)
    at me.nallar.tickthreading.minecraft.ThreadManager$1.run(ThreadManager.java:41)
    at me.nallar.tickthreading.minecraft.ThreadManager$ServerWorkThread.run(ThreadManager.java:167)

I keep getting this randomly should I include the whole log?

**Comment by LunNova** [comment]
Duplicate of #466.
